packages/utils/src/bufferUtils.ts

```diff
@@ -0,0 +1,11 @@
+export const reverseBuffer = (src: Buffer) => {
```

Comment:
file should be called `reverseBuffer.ts` we are doing function per file in this package

Reply:
Probably better to leave comments for cherrypicked commits in [Szymon's PR](https://github.com/trezor/trezor-suite/pull/6486)

Reply:
I see, this is confusing but I moved comments there.

Reply:
Since that PR is not merged and it's not final, better keep all discussions there 🙏 once this one is finalised, I will communicate with Szymon and make sure those commits can be merged with this PR if needed.